configuration/parse.go

```diff
@@ -35,24 +42,53 @@ func Parse(args []string) (Configuration, error) {
 	flag.Var(&config.CveList, "exclude-vulnerability", "Comma separated list of CVEs to exclude")
 	flag.StringVar(&excludeVulnerabilityFilePath, "exclude-vulnerability-file", "./.nancy-ignore", "Path to a file containing newline separated CVEs to be excluded")
 
+	iqCommand := flag.NewFlagSet("iq", flag.ExitOnError)
+	iqCommand.StringVar(&config.User, "user", "admin", "Specify username for request")
+	iqCommand.StringVar(&config.Token, "token", "admin123", "Specify token/password for request")
+	iqCommand.StringVar(&config.Server, "server-url", "http://localhost:8070", "Specify Nexus IQ Server URL/port")
+	iqCommand.StringVar(&config.Application, "application", "", "Specify application ID for request")
+	iqCommand.StringVar(&config.Stage, "stage", "develop", "Specify stage for application")
+
 	flag.Usage = func() {
-		_, _ = fmt.Fprintf(os.Stderr, "Usage: \nnancy [options] </path/to/Gopkg.lock>\nnancy [options] </path/to/go.sum>\n\nOptions:\n")
+		_, _ = fmt.Fprintf(os.Stderr, `Usage:
+	nancy [options] </path/to/Gopkg.lock>
+	nancy [options] </path/to/go.sum>
+	nancy iq [options]
```

Comment:
This is out  of date in that technically we can do this with just `go list -m all | nancy [options]` as well :) not really your problem its just that you touched it so now I noticed.

Reply:
In: https://github.com/sonatype-nexus-community/nancy/pull/68/commits/e46bd59d868878314a2cdcf1cff10ec1f144558f